### PR TITLE
Update the Readme to reflect the change in interface

### PR DESCRIPTION
**Pull request by frmsaul**

### CODE REVIEW

**Comment by vgm64** [comment]
What exactly are the cases where the api key is needed? @UnePierre do you know? Is it for notebook running only? To this day new maps I generate still load without issue. It'd be great to include messaging for when the api key is needed.

**Comment by frmsaul** [comment]
The need for the api key seems to appear when you embed the generated map in another "complex page". Not sure what their criteria for a "complex page" is exactly but it seems to with the marketing information [here](https://developers.google.com/maps/documentation/embed/). 

> It'd be great to include messaging for when the api key is needed.

That's an excellent idea. Let me look into it. 

Saul

**Comment by frmsaul** [comment]
Btw, thanks for creating this package! It's extremely useful.

File: README.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ make creating exploratory map views effortless. Here's a crash course:
 
     import gmplot
 
-    gmap = gmplot.GoogleMapPlotter(37.428, -122.145, 16)
+    gmap = gmplot.GoogleMapPlotter(37.428, -122.145, 16, apikey=YOUR_GOOGLE_API_KEY)
```
Comment (frslm) [inline README.rst:13]: I'd recommend defining the API key as `apikey = '' # (your API key goes here)` right above this line, then simply passing `apikey=apikey` into `gmplot.GoogleMapPlotter()`. That way users who copy-paste this whole code block won't get an error if they don't replace `YOUR_GOOGLE_API_KEY` (and a blank API key works, albeit with limited functionality).

File: README.rst [hunk 2]
```diff
@@ -19,6 +19,7 @@ make creating exploratory map views effortless. Here's a crash course:
 
     gmap.draw("mymap.html")
 
+You can get your google api key on the `google developers site <https://developers.google.com/maps/documentation/javascript/get-api-key>`_.
```
Comment (frslm) [inline README.rst:22]: `google api key` -> `Google Maps API key` or `Google API key`
`google developers site` -> `Google developers site`

**Pull request merged**